### DISCUSSION TITLE
Export report as PDF

### DISCUSSION THREAD

**Discussion by lakshaa** [globaleaks/globaleaks-whistleblowing-software#3733] [General]
Can a recipient export a report as PDF? Currently, we have to click on Print and then select as PDF. If we click on export, it is downloaded as a zip file and in .txt format. Any option to directly download it as PDF when clicking on the button "export"?

**Comment by evilaliv3** [comment]
Hello @lakshaa 

This is currently not possible but we will try to implement soon a replacement for the TXT export with a PDF export.
